### DISCUSSION TITLE
superset-frontend can not run

### DISCUSSION THREAD

**Discussion by crystal20277** [apache/superset#18355] [Q&A / Help]
My branch is master, and the nodejs and npm versions are：

npm --version && node --version
7.24.0
v16.10.0

First, compile with the command npm install, and then run the command npm run dev to report an error. The error information is as follows:

> superset@0.0.0dev dev
> webpack --mode=development --color `--watch`

[Superset Plugin] Use symlink source for @superset-ui/chart-controls @ ^0.18.25
[Superset Plugin] Use symlink source for @superset-ui/core @ ^0.18.25
[Superset Plugin] Use symlink source for @superset-ui/legacy-plugin-chart-calendar @ ^0.18.25
[Superset Plugin] Use symlink source for @superset-ui/legacy-plugin-chart-chord @ ^0.18.25
[Superset Plugin] Use symlink source for @superset-ui/legacy-plugin-chart-country-map @ ^0.18.25
[Superset Plugin] Use symlink source for @superset-ui/legacy-plugin-chart-event-flow @ ^0.18.25
[Superset Plugin] Use symlink source for @superset-ui/legacy-plugin-chart-force-directed @ ^0.18.25
[Superset Plugin] Use symlink source for @superset-ui/legacy-plugin-chart-heatmap @ ^0.18.25
[Superset Plugin] Use symlink source for @superset-ui/legacy-plugin-chart-histogram @ ^0.18.25
[Superset Plugin] Use symlink source for @superset-ui/legacy-plugin-chart-horizon @ ^0.18.25
[Superset Plugin] Use symlink source for @superset-ui/legacy-plugin-chart-map-box @ ^0.18.25
[Superset Plugin] Use symlink source for @superset-ui/legacy-plugin-chart-paired-t-test @ ^0.18.25
[Superset Plugin] Use symlink source for @superset-ui/legacy-plugin-chart-parallel-coordinates @ ^0.18.25
[Superset Plugin] Use symlink source for @superset-ui/legacy-plugin-chart-partition @ ^0.18.25
[Superset Plugin] Use symlink source for @superset-ui/legacy-plugin-chart-pivot-table @ ^0.18.25
[Superset Plugin] Use symlink source for @superset-ui/legacy-plugin-chart-rose @ ^0.18.25
[Superset Plugin] Use symlink source for @superset-ui/legacy-plugin-chart-sankey @ ^0.18.25
[Superset Plugin] Use symlink source for @superset-ui/legacy-plugin-chart-sankey-loop @ ^0.18.25
[Superset Plugin] Use symlink source for @superset-ui/legacy-plugin-chart-sunburst @ ^0.18.25
[Superset Plugin] Use symlink source for @superset-ui/legacy-plugin-chart-treemap @ ^0.18.25
[Superset Plugin] Use symlink source for @superset-ui/legacy-plugin-chart-world-map @ ^0.18.25
[Superset Plugin] Use symlink source for @superset-ui/legacy-preset-chart-big-number @ ^0.18.25
[Superset Plugin] Use symlink source for @superset-ui/legacy-preset-chart-nvd3 @ ^0.18.25
[Superset Plugin] Use symlink source for @superset-ui/plugin-chart-echarts @ ^0.18.25
[Superset Plugin] Use symlink source for @superset-ui/plugin-chart-pivot-table @ ^0.18.25
[Superset Plugin] Use symlink source for @superset-ui/plugin-chart-table @ ^0.18.25
[Superset Plugin] Use symlink source for @superset-ui/plugin-chart-word-cloud @ ^0.18.25
[Superset Plugin] Use symlink source for @superset-ui/preset-chart-xy @ ^0.18.25

740 assets
9632 modules

WARNING in ./node_modules/@data-ui/shared/esm/enhancer/WithTooltip.js 3:60-78
export 'withTooltipPropTypes' (imported as 'vxTooltipPropTypes') was not found in '@vx/tooltip/build/tooltips/TooltipWithBounds' (possible exports: __esModule, default)

WARNING in ./packages/superset-ui-chart-controls/src/components/Tooltip.tsx 23:0-48
export 'TooltipProps' (reexported as 'TooltipProps') was not found in 'antd/lib/tooltip' (possible exports: __esModule, default)

WARNING in ./packages/superset-ui-chart-controls/src/types.ts 1:295-338
export 'Metric' (reexported as 'Metric') was not found in '@superset-ui/core' (possible exports: AnnotationOpacity, AnnotationSourceType, AnnotationStyle, AnnotationType, ApiLegacy, ApiV1, AppSection, BRAND_COLOR, Behavior, BuildQue
ry, CategoricalAirbnb, CategoricalColorNamespace, CategoricalColorScale, CategoricalD3, CategoricalEcharts, CategoricalGoogle, CategoricalLyft, CategoricalPreset, CategoricalScheme, CategoricalSuperset, ChartClient, ChartDataProvide
r, ChartFrame, ChartMetadata, ChartPlugin, ChartProps, ColorSchemeRegistry, ComparisionType, DTTM_ALIAS, DatasourceKey, DatasourceType, EXTRA_FORM_DATA_APPEND_KEYS, EXTRA_FORM_DATA_OVERRIDE_EXTRA_KEYS, EXTRA_FORM_DATA_OVERRIDE_KEYS,
 EXTRA_FORM_DATA_OVERRIDE_REGULAR_KEYS, EXTRA_FORM_DATA_OVERRIDE_REGULAR_MAPPINGS, EmotionCacheProvider, ExtensibleFunction, FAST_DEBOUNCE, FeatureFlag, GenericDataType, LOCAL_PREFIX, NumberFormats, NumberFormatter, NumberFormatterR
egistry, OverwritePolicy, PREVIEW_TIME, PREVIEW_VALUE, PandasAxis, Plugin, Preset, QueryMode, Registry, RegistryWithDefaultKey, RollingType, SLOW_DEBOUNCE, SafeMarkdown, SequentialCommon, SequentialD3, SequentialScheme, SuperChart,
SupersetApiError, SupersetApiErrorType, SupersetClient, SupersetClientClass, ThemeProvider, TimeFormats, TimeFormatter, TimeGranularity, TooltipFrame, TooltipTable, WithLegend, __hack_reexport_AdvancedAnalytics, __hack_reexport_Colu
mn, __hack_reexport_Datasource, __hack_reexport_Metric, __hack_reexport_Query, __hack_reexport_QueryFormData, __hack_reexport_QueryResponse, __hack_reexport_Time, __hack_reexport_chart_Base, __hack_reexport_chart_QueryResponse, __ha
ck_reexport_chart_TransformFunction, __hack_reexport_connection, __hack_reexport_dimension, __hack_reexport_jed, __hack_reexport_trasnslation, addLocaleData, addTranslation, addTranslations, buildQueryContext, buildQueryObject, call
Api, computeMaxFontSize, configure, convertFilter, convertKeysToCamelCase, createD3NumberFormatter, createD3TimeFormatter, createDurationFormatter, createEmotionCache, createLoadableRenderer, createMultiFormatter, createSiAtMostNDig
itFormatter, createSmartNumberFormatter, css, defineSharedModule, defineSharedModules, emotionCache, ensureIsArray, ensureIsInt, evalExpression, extractTimegrain, formatNumber, formatTime, formatTimeRange, getCategoricalSchemeRegist
ry, getChartBuildQueryRegistry, getChartComponentRegistry, getChartControlPanelRegistry, getChartMetadataRegistry, getChartTransformPropsRegistry, getColumnLabel, getContrastingColor, getMetricLabel, getMultipleTextDimensions, getNu
mberFormatter, getNumberFormatterRegistry, getSequentialSchemeRegistry, getTextDimension, getTimeFormatter, getTimeFormatterForGranularity, getTimeFormatterRegistry, getTimeRangeFormatter, isAdhocMetricSimple, isBinaryAdhocFilter, i
sBinaryOperator, isDefined, isDruidFormData, isEventAnnotationLayer, isFeatureEnabled, isFormulaAnnotationLayer, isIntervalAnnotationLayer, isPhysicalColumn, isPostProcessingAggregation, isPostProcessingBoxplot, isPostProcessingComp
are, isPostProcessingContribution, isPostProcessingCum, isPostProcessingDiff, isPostProcessingPivot, isPostProcessingProphet, isPostProcessingResample, isPostProcessingRolling, isPostProcessingSort, isRecordAnnotationResult, isRequi
red, isSavedMetric, isSetAdhocFilter, isSetOperator, isSimpleAdhocFilter, isTableAnnotationLayer, isTimeseriesAnnotationLayer, isTimeseriesAnnotationResult, isUnaryAdhocFilter, isUnaryOperator, isValidExpression, jsx, legacyValidate
Integer, legacyValidateNumber, logging, makeApi, makeSingleton, mergeMargin, normalizeOrderBy, parseLength, promiseTimeout, reactify, removeDuplicates, reset, resetTranslation, seed, seedRandom, smartDateDetailedFormatter, smartDate
Formatter, smartDateVerboseFormatter, styled, supersetTheme, t, tn, useTheme, validateInteger, validateNonEmpty, validateNumber, withTheme)

WARNING in ./packages/superset-ui-chart-controls/src/types.ts 2:0-63
export 'ControlFormItemSpec' (reexported as 'ControlFormItemSpec') was not found in './components/ControlForm' (possible exports: ControlFormItem, ControlFormItemComponents, ControlFormRow, default)

WARNING in ./packages/superset-ui-chart-controls/src/types.ts 3:0-75
export 'ControlComponentProps' (reexported as 'ControlComponentProps') was not found in './shared-controls/components/types' (module has no exports)

WARNING in ./packages/superset-ui-core/src/chart/index.ts 31:0-75
export 'BuildQuery' (reexported as 'BuildQuery') was not found in './registries/ChartBuildQueryRegistrySingleton' (possible exports: default)

WARNING in ./src/common/components/index.tsx 29:0-62
export 'ModalProps' (reexported as 'ModalProps') was not found in 'antd/lib/modal' (possible exports: __esModule, default)

WARNING in ./src/common/components/index.tsx 30:0-45
export 'FormInstance' (reexported as 'FormInstance') was not found in 'antd/lib/form' (possible exports: __esModule, default)

WARNING in ./src/common/components/index.tsx 31:0-50
export 'RadioChangeEvent' (reexported as 'RadioChangeEvent') was not found in 'antd/lib/radio' (possible exports: Button, Group, __esModule, default)

WARNING in ./src/common/components/index.tsx 32:0-42
export 'TreeProps' (reexported as 'TreeProps') was not found in 'antd/lib/tree' (possible exports: __esModule, default)

WARNING in ./src/common/components/index.tsx 33:0-62
export 'AlertProps' (reexported as 'AlertProps') was not found in 'antd/lib/alert' (possible exports: __esModule, default)

WARNING in ./src/common/components/index.tsx 34:0-65
export 'SelectProps' (reexported as 'SelectProps') was not found in 'antd/lib/select' (possible exports: __esModule, default)

WARNING in ./src/common/components/index.tsx 35:0-63
export 'ListItemProps' (reexported as 'ListItemProps') was not found in 'antd/lib/list' (possible exports: ListConsumer, ListContext, __esModule, default)

WARNING in ./src/components/Form/Form.tsx 33:0-21
export 'FormProps' (reexported as 'FormProps') was not found in 'antd/lib/form' (possible exports: __esModule, default)

WARNING in ./src/components/Icons/index.tsx 162:0-20
export 'default' (reexported as 'IconType') was not found in './IconType' (module has no exports)

WARNING in ./src/components/Switch/index.tsx 28:0-23
export 'SwitchProps' (reexported as 'SwitchProps') was not found in 'antd/lib/switch' (possible exports: __esModule, default)

WARNING in ./src/featureFlags.ts 1:295-359
export 'FeatureFlagMap' (reexported as 'FeatureFlagMap') was not found in '@superset-ui/core' (possible exports: AnnotationOpacity, AnnotationSourceType, AnnotationStyle, AnnotationType, ApiLegacy, ApiV1, AppSection, BRAND_COLOR, Be
havior, BuildQuery, CategoricalAirbnb, CategoricalColorNamespace, CategoricalColorScale, CategoricalD3, CategoricalEcharts, CategoricalGoogle, CategoricalLyft, CategoricalPreset, CategoricalScheme, CategoricalSuperset, ChartClient,
ChartDataProvider, ChartFrame, ChartMetadata, ChartPlugin, ChartProps, ColorSchemeRegistry, ComparisionType, DTTM_ALIAS, DatasourceKey, DatasourceType, EXTRA_FORM_DATA_APPEND_KEYS, EXTRA_FORM_DATA_OVERRIDE_EXTRA_KEYS, EXTRA_FORM_DAT
A_OVERRIDE_KEYS, EXTRA_FORM_DATA_OVERRIDE_REGULAR_KEYS, EXTRA_FORM_DATA_OVERRIDE_REGULAR_MAPPINGS, EmotionCacheProvider, ExtensibleFunction, FAST_DEBOUNCE, FeatureFlag, GenericDataType, LOCAL_PREFIX, NumberFormats, NumberFormatter,
NumberFormatterRegistry, OverwritePolicy, PREVIEW_TIME, PREVIEW_VALUE, PandasAxis, Plugin, Preset, QueryMode, Registry, RegistryWithDefaultKey, RollingType, SLOW_DEBOUNCE, SafeMarkdown, SequentialCommon, SequentialD3, SequentialSche
me, SuperChart, SupersetApiError, SupersetApiErrorType, SupersetClient, SupersetClientClass, ThemeProvider, TimeFormats, TimeFormatter, TimeGranularity, TooltipFrame, TooltipTable, WithLegend, __hack_reexport_AdvancedAnalytics, __ha
ck_reexport_Column, __hack_reexport_Datasource, __hack_reexport_Metric, __hack_reexport_Query, __hack_reexport_QueryFormData, __hack_reexport_QueryResponse, __hack_reexport_Time, __hack_reexport_chart_Base, __hack_reexport_chart_Que
ryResponse, __hack_reexport_chart_TransformFunction, __hack_reexport_connection, __hack_reexport_dimension, __hack_reexport_jed, __hack_reexport_trasnslation, addLocaleData, addTranslation, addTranslations, buildQueryContext, buildQ
ueryObject, callApi, computeMaxFontSize, configure, convertFilter, convertKeysToCamelCase, createD3NumberFormatter, createD3TimeFormatter, createDurationFormatter, createEmotionCache, createLoadableRenderer, createMultiFormatter, cr
eateSiAtMostNDigitFormatter, createSmartNumberFormatter, css, defineSharedModule, defineSharedModules, emotionCache, ensureIsArray, ensureIsInt, evalExpression, extractTimegrain, formatNumber, formatTime, formatTimeRange, getCategor
icalSchemeRegistry, getChartBuildQueryRegistry, getChartComponentRegistry, getChartControlPanelRegistry, getChartMetadataRegistry, getChartTransformPropsRegistry, getColumnLabel, getContrastingColor, getMetricLabel, getMultipleTextD
imensions, getNumberFormatter, getNumberFormatterRegistry, getSequentialSchemeRegistry, getTextDimension, getTimeFormatter, getTimeFormatterForGranularity, getTimeFormatterRegistry, getTimeRangeFormatter, isAdhocMetricSimple, isBina
ryAdhocFilter, isBinaryOperator, isDefined, isDruidFormData, isEventAnnotationLayer, isFeatureEnabled, isFormulaAnnotationLayer, isIntervalAnnotationLayer, isPhysicalColumn, isPostProcessingAggregation, isPostProcessingBoxplot, isPo
stProcessingCompare, isPostProcessingContribution, isPostProcessingCum, isPostProcessingDiff, isPostProcessingPivot, isPostProcessingProphet, isPostProcessingResample, isPostProcessingRolling, isPostProcessingSort, isRecordAnnotatio
nResult, isRequired, isSavedMetric, isSetAdhocFilter, isSetOperator, isSimpleAdhocFilter, isTableAnnotationLayer, isTimeseriesAnnotationLayer, isTimeseriesAnnotationResult, isUnaryAdhocFilter, isUnaryOperator, isValidExpression, jsx
, legacyValidateInteger, legacyValidateNumber, logging, makeApi, makeSingleton, mergeMargin, normalizeOrderBy, parseLength, promiseTimeout, reactify, removeDuplicates, reset, resetTranslation, seed, seedRandom, smartDateDetailedForm
atter, smartDateFormatter, smartDateVerboseFormatter, styled, supersetTheme, t, tn, useTheme, validateInteger, validateNonEmpty, validateNumber, withTheme)

WARNING in ./src/utils/textUtils.ts 23:13-46
Module not found: Error: Can't resolve '../../../superset_text' in 'D:\workspace\python_workspace\superset_master\superset-frontend\src\utils'

ERROR in ./plugins/legacy-plugin-chart-calendar/src/ReactCalendar.jsx 27:2
Module parse failed: Unexpected token (27:2)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|
| const Calender = ({ className, ...otherProps }) => (
>   <div className={className}>
|     <ReactComponent {...otherProps} />
|   </div>

ERROR in ./plugins/legacy-plugin-chart-chord/src/ReactChord.jsx 27:2
Module parse failed: Unexpected token (27:2)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|
| const Chord = ({ className, ...otherProps }) => (
>   <div className={className}>
|     <ReactComponent {...otherProps} />
|   </div>

ERROR in ./plugins/legacy-plugin-chart-histogram/src/Histogram.jsx 84:6
Module parse failed: Unexpected token (84:6)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|
|     return (
>       <WithLegend
|         className={`superset-legacy-chart-histogram ${className}`}
|         width={width}

ERROR in ./plugins/legacy-plugin-chart-horizon/src/HorizonChart.jsx 86:6
Module parse failed: Unexpected token (86:6)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|
|     return (
>       <div
|         className={`superset-legacy-chart-horizon ${className}`}
|         style={{ height }}

ERROR in ./plugins/legacy-plugin-chart-map-box/src/MapBox.jsx 123:6
Module parse failed: Unexpected token (123:6)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|
|     return (
>       <MapGL
|         {...viewport}
|         mapStyle={mapStyle}

ERROR in ./plugins/legacy-plugin-chart-paired-t-test/src/PairedTTest.jsx 48:6
Module parse failed: Unexpected token (48:6)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|
|     return (
>       <div className={`superset-legacy-chart-paired-t-test ${className}`}>
|         <div className="paired-ttest-table">
|           <div className="scrollbar-content">

ERROR in ./plugins/legacy-plugin-chart-parallel-coordinates/src/ReactParallelCoordinates.jsx 27:2
Module parse failed: Unexpected token (27:2)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|
| const ParallelCoordianes = ({ className, ...otherProps }) => (
>   <div className={className}>
|     <ReactComponent {...otherProps} />
|   </div>

ERROR in ./plugins/legacy-plugin-chart-sankey/src/ReactSankey.jsx 27:2
Module parse failed: Unexpected token (27:2)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|
| const SankeyComponent = ({ className, ...otherProps }) => (
>   <div className={className}>
|     <ReactSanKey {...otherProps} />
|   </div>

ERROR in ./plugins/legacy-plugin-chart-world-map/src/ReactWorldMap.jsx 27:2
Module parse failed: Unexpected token (27:2)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|
| const WorldMapComponent = ({ className, ...otherProps }) => (
>   <div className={className}>
|     <ReactWorldMap {...otherProps} />
|   </div>

ERROR in ./plugins/legacy-preset-chart-nvd3/src/ReactNVD3.jsx 37:2
Module parse failed: Unexpected token (37:2)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|
| const NVD3 = ({ className, ...otherProps }) => (
>   <div className={className}>
|     <ReactNVD3 {...otherProps} />
|   </div>

ERROR in ./src/SqlLab/App.jsx 128:2
Module parse failed: Unexpected token (128:2)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|
| const Application = () => (
>   <Provider store={store}>
|     <ThemeProvider theme={theme}>
|       <App />

ERROR in ./src/components/CopyToClipboard/index.jsx 39:12
Module parse failed: Unexpected token (39:12)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|
| const defaultProps = {
>   copyNode: <span>Copy</span>,
|   onCopyEnd: () => {},
|   shouldShowText: true,

ERROR in ./src/components/Datasource/DatasourceEditor.jsx 122:2
Module parse failed: Unexpected token (122:2)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|
| const checkboxGenerator = (d, onChange) => (
>   <CheckboxControl value={d} onChange={onChange} />
| );
| const DATA_TYPES = [

ERROR in ./src/components/ErrorBoundary/index.jsx 50:8
Module parse failed: Unexpected token (50:8)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|       const firstLine = error.toString();
|       const message = (
>         <span>
|           <strong>{t('Unexpected error')}</strong>
|           {firstLine ? `: ${firstLine}` : ''}

ERROR in ./src/dashboard/components/Dashboard.jsx 291:13
Module parse failed: Unexpected token (291:13)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|   render() {
|     if (this.context.loading) {
>       return <Loading />;
|     }
|     return (

ERROR in ./src/dashboard/components/PropertiesModal/index.jsx 352:6
Module parse failed: Unexpected token (352:6)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|
|     return (
>       <Row gutter={16}>
|         <Col xs={24} md={12}>
|           <h3 style={{ marginTop: '1em' }}>{t('Access')}</h3>

ERROR in ./src/explore/components/ControlHeader.jsx 53:8
Module parse failed: Unexpected token (53:8)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|     if (this.props.hovered) {
|       return (
>         <span
|           css={theme => css`
|             position: absolute;

ERROR in ./src/explore/components/controls/TextAreaControl.jsx 74:8
Module parse failed: Unexpected token (74:8)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|       }
|       return (
>         <TextAreaEditor
|           mode={this.props.language}
|           style={style}

ERROR in ./src/explore/controls.jsx 130:23
Module parse failed: Unexpected token (130:23)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|       'to limit the number of fetched and rendered series.',
|   ),
>   optionRenderer: c => <StyledColumnOption column={c} showType />,
|   valueKey: 'column_name',
|   filterOption: ({ data: opt }, text) =>

ERROR in ./src/explore/index.jsx 42:16
Module parse failed: Unexpected token (42:16)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| );
|
> ReactDOM.render(<App store={store} />, document.getElementById('app'));
|

ERROR in ./src/showSavedQuery/index.jsx 39:4
Module parse failed: Unexpected token (39:4)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|
|   ReactDom.render(
>     <div>
|       <Form
|         schema={config.JSONSCHEMA}

ERROR in ./src/visualizations/FilterBox/FilterBox.jsx 265:8
Module parse failed: Unexpected token (265:8)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|     if (showDateFilter) {
|       return (
>         <div className="row space-1">
|           <div
|             className="col-lg-12 col-xs-12"

ERROR in ./src/visualizations/FilterBox/controlPanel.jsx 73:9
Module parse failed: Unexpected token (73:9)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|           },
|         ],
>         [<hr />],
|         [
|           {

ERROR in ./src/visualizations/TimeTable/TimeTable.jsx 118:10
Module parse failed: Unexpected token (118:10)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|         },
|         Header: () => (
>           <>
|             {columnConfig.label}{' '}
|             {columnConfig.tooltip && (

webpack 5.52.1 compiled with 24 errors and 18 warnings in 69492 ms

**Comment by rusackas** [comment]
There have been a lot of changes taking place following the move to make Superset a monorepo. Can you please verify if this is still an issue?

**Comment by CodeingBoy** [comment]
Are you running this on Windows? `webpack` seems not running very well on Windows. I had also encounter this and after switching to WSL it works well.

**Comment by qiangxue1206** [comment]
+1

**Comment by qiangxue1206** [comment]
![image](https://user-images.githubusercontent.com/44155933/147814483-8dd784e6-0ccc-4649-9bd0-9c4a222ed113.png)

**Comment by qiangxue1206** [comment]
> ![图片](https://user-images.githubusercontent.com/44155933/147814483-8dd784e6-0ccc-4649-9bd0-9c4a222ed113.png)

Add this can solve.

**Reply by leezj-cbm** [reply]
This helped. Thanks!

**Comment by dandanwy** [comment]
+1   window  cannot run

**Comment by slieer** [comment]
**npm run dev-server -- --port=9001**

WARNING in ./node_modules/@data-ui/shared/esm/enhancer/WithTooltip.js 3:60-78
**export 'withTooltipPropTypes' (imported as 'vxTooltipPropTypes') was not found in '@vx/tooltip/build/tooltips/TooltipWithBounds' (possible exports: __esModule, default)**

**WARNING in ./src/dashboard/components/Header/index.jsx 61:8-35
export 'UserWithPermissionsAndRoles' (imported as 'UserWithPermissionsAndRoles') was not found in 'src/types/bootstrapTypes' (module has no exports)**

**WARNING in ./src/components/Checkbox/index.tsx 19:0-74
export 'CheckboxProps' (reexported as 'CheckboxProps') was not found in 'src/components/Checkbox/Checkbox' (possible exports: default)**

**WARNING in ./src/utils/textUtils.ts 23:13-46
Module not found: Error: Can't resolve '../../../superset_text' in '/opt/workspace/superset/superset-frontend/src/utils'**

webpack 5.52.1 compiled with 4 warnings in 146495 ms
99% end closing watch compilation

root@R620:/opt/workspace/superset/superset-frontend# **npm version**
{
  superset: '0.0.0dev',
  npm: '7.24.2',
  node: '16.13.0',
  v8: '9.4.146.19-node.13',
  uv: '1.42.0',
  zlib: '1.2.11',
  brotli: '1.0.9',
  ares: '1.17.2',
  modules: '93',
  nghttp2: '1.45.1',
  napi: '8',
  llhttp: '6.0.4',
  openssl: '1.1.1l+quic',
  cldr: '39.0',
  icu: '69.1',
  tz: '2021a',
  unicode: '13.0',
  ngtcp2: '0.1.0-DEV',
  nghttp3: '0.1.0-DEV'
}

root@R620:/opt/workspace/superset/superset-frontend# **git config -l**
core.repositoryformatversion=0
core.filemode=true
core.bare=false
core.logallrefupdates=true
remote.origin.url=https://github.com/apache/superset
remote.origin.fetch=+refs/heads/*:refs/remotes/origin/*
branch.master.remote=origin
branch.master.merge=refs/heads/master
root@R620:/opt/workspace/superset/superset-frontend# git status
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean

root@R620:/opt/workspace/superset/superset-frontend# **git log -1**
commit 00eb6b1f5711e1af0b8dcfc05346ad0dacadddda (HEAD -> master, origin/master, origin/HEAD)
Author: Emmanuel Bavoux <33355870+opus-42@users.noreply.github.com>
Date:   Thu Feb 10 02:09:20 2022 +0100

    feat: editable title xl certified badge (#18626)

    * Add size to EditableTitle certified badge

    * Lint EditableTitle file

    Co-authored-by: Emmanuel Bavoux <emmanuel.bavoux@free2move.com>

**Comment by ruijielou** [comment]
If you are using VS Code, please add in the settings
```json
   "eslint.workingDirectories": [
        "superset-frontend"
    ]
```

then

```
npm run dev-server
```

**Comment by rusackas** [comment]
It's been a couple years since this thread saw any action, so I'll assume all is well with the current codebase. Holler if this needs reopening!

**Reply by rparsonsbb** [reply]
running off the docker compose off master (`55f18a3`) this actually just popped up on me

ran `docker compose build superset-node --no-cache` because I didn't expect the build process to take 20 minutes (curse you windows/docker/wsl disk performance!)

```
superset_node  | <s> [webpack.Progress] 99% done plugins IdleFileCachePlugin
superset_node  | <s> [webpack.Progress] 99% done plugins
superset_node  | <s> [webpack.Progress] 99%
superset_node  |
superset_node  | 1082 assets
superset_node  | 12333 modules
superset_node  |
superset_node  | WARNING in ./src/utils/textUtils.ts 21:11-44
superset_node  | Module not found: Error: Can't resolve '../../../superset_text' in '/app/superset-frontend/src/utils'
superset_node  |
superset_node  | webpack 5.94.0 compiled with 1 warning in 520327 ms
superset_node  | <s> [webpack.Progress] 99% cache store build dependencies
superset_node  | <s> [webpack.Progress] 99% cache store build dependencies IdleFileCachePlugin
superset_node  | <s> [webpack.Progress] 99% cache store build dependencies
```
